.github/workflows/ci.yml

```
@@ -175,6 +176,7 @@ jobs:
       rust: ${{ matrix.rust }}
       rust-target: ${{ matrix.platform.rust-target }}
       MSRV: ${{ needs.resolve.outputs.MSRV }}
+      verbose: ${{ needs.resole.outputs.verbose }}
```

```suggestion
      verbose: ${{ needs.resolve.outputs.verbose }}
```
small typo, also below